package.json

```diff
@@ -6,12 +6,16 @@
     "android": "react-native run-android",
     "ios": "react-native run-ios",
     "start": "react-native start",
+    "run-android:dev": "ENVFILE=.env react-native run-android --variant=devDebug --appIdSuffix=dev",
```

Comment:
What about simple 'android:dev'

Reply:
yep we can remove run- and keep only android:dev the same for others